py/desispec/scripts/proc.py

```diff
@@ -273,7 +282,44 @@ def main(args=None, comm=None):
 
         timer.stop('nightlybias')
 
-    #- this might be just nightly bias, with no single exposure to process
+    #-------------------------------------------------------------------------
+    #- Create cte model from several LED exposures
+    if args.nightlycte:
+
+        missing_ctecorrnight = 0
+
+        if rank == 0: # nightlycte is still not MPI so only rank 0 will do it
+
+            timer.start('nightlycte')
+
+            camword = create_camword(args.cameras)
+            cmd = f"desi_fit_cte_night -n {args.night} -c {camword}"
+
+            log.info(f'RUNNING {cmd}')
+
+            #- Note: nightly_bias may not produce all biasnight files if some
+            #- are determined to be worse than the default, so check existence
+            #- of output files separately.
```

Comment:
Is this comment vestigial?  nightly_bias seems irrelevant here, though I admit to limited understanding of what proc.py is trying to do.